### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by dimasciput**
- Using tab : 
  feti
- On mobile : 
  feti

### CODE REVIEW

**Comment by dimasciput** [comment]
Update search:

feti

File: django_project/feti/static/feti/js/scripts/views/map.js [hunk 1]
```diff
@@ -216,9 +228,34 @@ define([
                 }
             }
         },
+        addAddLayerToModeLayer: function (layer) {
```
Comment (meomancer) [inline django_project/feti/static/feti/js/scripts/views/map.js]: hi @dimasciput 
not sure this naming convention correct?

Comment (dimasciput) [reply django_project/feti/static/feti/js/scripts/views/map.js]: Thanks, there are still couple of codes need to be improved.

**Comment by dimasciput** [comment]
@meomancer could you test this PR? Thanks.

**Comment by meomancer** [comment]
LGTM for layout

@dimasciput 
but got issue
i get autocomplete, and try one of this, but no result when search
example "ABET TRAINING" for course

and also for campus, could you make provider as autocomplete?
example "University Of Stellenbosch", it is in result, but isn't presented on autocomplete

**Comment by cchristelis** [comment]
Thanks @dimasciput for this update. I would like to confirm that when viewing the site on a mobile device:
-  is the map the with of the device? 
-  where is the results overlay?

**Comment by dimasciput** [comment]
Hi @cchristelis , for mobile layout I only update the tabs view in this PR. For other view like results overlay I will create it in another PR.

**Comment by dimasciput** [comment]
@meomancer , thanks for the feedback. I will create another ticket for this.

**Pull request merged**